lhotse/recipes/aishell4.py

```diff
@@ -144,10 +144,8 @@ def prepare_aishell4(
         validate_recordings_and_supervisions(recording_set, supervision_set)
 
         if output_dir is not None:
-            supervision_set.to_file(
-                output_dir / f"aishell4_supervisions_{part}.jsonl.gz"
-            )
-            recording_set.to_file(output_dir / f"aishell4_recordings_{part}.jsonl.gz")
+            supervision_set.to_file(output_dir / f"supervisions_{part}.jsonl.gz")
```

Comment:
I think it is intentional to use `aishell4_supervions_xxx.jsonl.gz` instead of `supervisions_xxx.jsonl.gz`.

Please modify your code to add `prefix` and `suffix` to `read_manifests_if_cached()`.

Reply:
Oh, I noticed that the Lhotse recipes has been modified comparing with before. (All add the prefix.)

Reply:
Thanks.